.github/instructions/background-jobs.instructions.md

```
@@ -27,5 +27,11 @@ description: Background job architecture and task management patterns
 - Handle all RQ status values explicitly to avoid infinite polling
 - Use `cancelInProgress` flag to prevent polling interference during cancellation
 
+## Superuser Requirement for Background Jobs
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)